components/automate-ui/src/app/services/projects-filter/projects-filter.effects.ts

```diff
@@ -45,17 +45,16 @@ export class ProjectsFilterEffects {
       });
     })
   );
+  });
 
-  @Effect()
-  loadOptions$ = this.actions$.pipe(
+  loadOptions$ = createEffect(() => this.actions$.pipe(
     ofType<LoadOptions>(ProjectsFilterActionTypes.LOAD_OPTIONS),
-    mergeMap(this.loadOptionsAction$()));
+    mergeMap(this.loadOptionsAction$())));
 
-  @Effect({ dispatch: false })
-  saveOptions$ = this.actions$.pipe(
+  saveOptions$ = createEffect(() => this.actions$.pipe(
     ofType<SaveOptions>(ProjectsFilterActionTypes.SAVE_OPTIONS),
     tap(({ payload }) => this.projectsFilter.storeOptions(payload))
-  );
+    ), { dispatch: false });
```

Comment:
Why was this `dispatch` added? What does it do?

Reply:
We need to use `{dispatch: false}` whenever our effect isn't returning an action.  In this case, dispatch wasn't _added_ here, it already existed but it now looks a little different in the new style of writing effects.

In the old format it was composed as
```
@Effect( {dispatch: false} )
```

in the new style
```
createEffect( () => some code, {dispatch: false})
```